aptos-move/move-examples/fungible_asset/sources/coin_example.move

```diff
@@ -0,0 +1,136 @@
+module aptos_framework::coin_example {
+    use aptos_framework::object;
+    use std::string::String;
+    use aptos_framework::object::{generate_extend_ref, Object, disable_ungated_transfer, generate_transfer_ref};
+    use fungible_asset::managed_fungible_asset::{initialize_managing_capabilities, mint_by_asset_owner, freeze_by_asset_owner, burn_by_asset_owner};
+    use std::signer::address_of;
+    use fungible_asset::fungible_asset;
+    use std::error;
+    use aptos_std::smart_table;
+    use aptos_std::smart_table::SmartTable;
+    #[test_only]
+    use std::string;
+    #[test_only]
+    use aptos_framework::account::create_account_for_test;
+
+    struct Coin has key {
```

Comment:
I kinda wanna see an argument for why this shouldn't all be in FungibleSource. I don't see the advantage of having an extension to FungibleSource and I really want to kill coin :P

Reply:
It can be, I just make this one an asset to show how to use that. 
After a second thought, In fact, the asset doesn't have to exist actually. I will refactor this part tmr.

Reply:
Yeah +1 on merging this with FungibleSource